tests/capture/autograph/test_autograph.py

```diff
@@ -311,22 +312,105 @@ def circ():
         plxpr = qml.capture.make_plxpr(circ)()
         assert jax.core.eval_jaxpr(plxpr.jaxpr, plxpr.consts)[0] == -1
 
-    def test_ctrl_op(self):
-        """Test that controlled operators successfully pass through autograph"""
+    def test_adjoint_of_operator_type(self):
+        """Test that the adjoint of an operator successfully passes through autograph"""
 
         @qml.qnode(qml.device("default.qubit", wires=2))
         def circ():
-            qml.X(1)
-            qml.ctrl(qml.X(0), 1)
+            qml.adjoint(qml.X)(0)
             return qml.expval(qml.Z(0))
 
-        plxpr = qml.capture.make_plxpr(circ, autograph=True)()
+        plxpr = qml.capture.make_plxpr(circ)()
```

Comment:
If the test is for autograph then I would prefer that we explicitly specify `autograph=True`. It'll future proof us in case we ever change the default.